include/gatekeeper_gt.h

```diff
@@ -126,7 +130,8 @@ struct gt_config {
 
 struct gt_config *alloc_gt_conf(void);
 int gt_conf_put(struct gt_config *gt_conf);
-int run_gt(struct net_config *net_conf, struct gt_config *gt_conf);
+int run_gt(const char *policy_dir_name, const char *gt_config_file_name,
+	struct net_config *net_conf, struct gt_config *gt_conf);
```

Comment:
It looks cleaner to me to put the important parameters such as `net_conf` and `gt_conf` first and to put supplementary parameters after, but I don't have a strong argument so feel free to ignore if you disagree.